### PR TITLE
Change Feedback styling to pure css

### PR DESCRIPTION
**Pull request by rmccar**
### What is the context of this PR?
This change is to remove the need for an svg to help style the Feedback component and use pure css instead

### How to review 
Check Feedback looks as expected

### CODE REVIEW

**Comment by rmccar** [comment]
[Convert feedback component border to css only](https://trello.com/c/tI2l4M9j/732-convert-feedback-component-border-to-css-only)

**Comment by boxadesign** [comment]
image

Do we need a `div` here?

**Comment by jrbarnes9** [comment]
Need to be able to change heading tag level.

File: src/components/feedback/_macro.njk [hunk 1]
```diff
@@ -1,18 +1,8 @@
...
-        </div>
-        <div class="feedback__footer">
-            <svg class="svg-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 25.26 25.28">
-                <polyline points="25.26 0 21 0 21 0 3 18 3 0 0 0 0 25.28"></polyline>
-                <polygon points="3 0 21 0 3 18 3 0" style="fill:#fff"></polygon>
-            </svg>
-        </div>
+        <h{{ headerSize }}>{{ params.header }}</h{{ headerSize }}>
+        <p>{{ params.content }}</p>
+        <a href="{{ params.url }}">{{- params.linkText -}}</a>
```
Comment (jrbarnes9) [inline src/components/feedback/_macro.njk:6]: I think this link was bold in Joe's design?

**Review by boxadesign — APPROVED** [review]

**Pull request merged**